### PR TITLE
[ALLUXIO-2166]Add a debug message in SwiftUnderFileSystem

### PR DESCRIPTION
**Pull request by greathjt**
Add a debug message in SwiftUnderFileSystem.java::listRecursive,detail:
https://alluxio.atlassian.net/browse/ALLUXIO-2166

### CODE REVIEW

File: underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java [hunk 1]
```diff
@@ -303,6 +303,7 @@ public boolean isFile(String path) throws IOException {
 
   @Override
   public String[] listRecursive(String path) throws IOException {
+	LOG.debug("List {} recursively", path);
```
Comment (uronce-cc) [inline underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java]: please fix the wrong indentation, code style matters.

**Comment by yupeng9** [comment]
LGTM

**Comment by yupeng9** [comment]
jenkins, test this please

**Comment by RongGu** [comment]
LGTM.

**Comment by uronce-cc** [comment]
LGTM

**Pull request merged**